README.txt

```diff
@@ -1,162 +1,89 @@
-+
-Users in roles with the 'view vote up/down count on nodes' permission
-will be able to see the widget for casting votes on nodes.
-+
-*** vud_term
-**** administer vote up/down on terms
-+
-Users in roles with the 'administer vote up/down on terms' permission
-will be able to modify the settings regarding voting on terms.
-+
-**** use vote up/down on terms
-+
-Users in roles with the 'use vote up/down on terms' permission will be
-able to see the widget for casting votes on terms.
-+
-* Customize the  settings in Administer >> Site configuration >>
-Vote Up/Down, so you can
-** match:
-*** vud_node: what node types should be votable
-*** vud_comment: what comments of node types should be votable
-*** vid_term: what terms of node types should be votable
-** which widget is going to be used
-** if you want to display the widget in on teasers/full nodes
-** if the user is going to be able to reset their own votes
-* Start voting!
+CONFIGURATION
+-------------
 
-== Customization ==
+ * Configure permissions in Administer >> People >> Permissions
+   >> Vote Up/Down:
 
-You can write your own widget for Vote Up/Down, and you can put it in
-your module or in your theme. Please take a look to
-link:WIDGETAPI.html[WIDGETAPI.txt] for more information.
+  - access vote up/down statistics:
+     Users in roles with the 'access vote up/down statistics' permission
+     will be able to see the votes performed by each user on its 'Votes' tab.
 
-=== Theming widgets and votes ===
+  - administer vote up/down:
+     Users in roles with the 'administer vote up/down' permission will be able
+     to modify the Voting API tag for Vote Up/Down votes.
 
-Since widgets are implemented through plugins, and we have many plugins
-to choose at runtime, we can not use direct theme templates, so instead
-we render by hands the templates.
+  - reset vote up/down votes:
+     Users in roles with the 'reset vote up/down votes' permission will be able
+     to undo their own votes if it's also permitted in the configuration for
+     the respective module.
 
-NOTE: It is not possible to decide dinamically the path where the
-template is located. In contrast we can dinamically define function and
-template names.
+  - use vote up/down:
+     Users in roles with the 'use vote up/down' permission will be able to
+     actually cast a vote with vote up/down(for the callback).
 
-So, before rendering the template('widget.tpl.php' or 'votes.tpl.php') we
-verify in the following order the files:
+ * Start voting!
 
-* For 'vud_comment'
-+
-----
-<template_type>_comment_<plugin_name>__<node_type>
-<template_type>_comment_<plugin_name>
-<template_type>_comment
-----
-* For 'vud_node'
-+
-----
-<template_type>_comment_<plugin_name>__<node_type>
-<template_type>_comment_<plugin_name>
-<template_type>_comment
-----
-* For 'vud_term'
-+
-----
-<template_type>_comment_<plugin_name>__<term_vid>
-<template_type>_comment_<plugin_name>
-<template_type>_comment
-----
+CUSTOMIZATION
```

Comment:
This section may need some changes, but let's defer that for when we do have a plugin implementation in D8.